mslib/mscolab/sockets_manager.py

```diff
@@ -213,14 +275,15 @@ def handle_file_save(self, json_req):
         op_id = json_req['op_id']
         content = json_req['content']
         comment = json_req.get('comment', "")
+        messageText = json_req.get('messageText')
```

Comment:
This should have really been called `message_text`. Unfortunately, it seems like flake8 doesn't check for naming conventions...